.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: required
 services:
 - docker
 scala:
-- 2.12.10
+- 2.13.1
```

Comment:
You also need to change the conditions below: `"$TRAVIS_SCALA_VERSION" = "2.12.10"`